infrastructure/zk/src/config.ts

```diff
@@ -151,7 +151,7 @@ export function pushConfig(environment?: string, diff?: string) {
         false
     );
 
-    env.modify('DATABASE_URL', `postgres://postgres@localhost/${environment}`, l2InitFile, false);
+    env.modify('DATABASE_URL', `postgres://postgres:notsecurepassword@localhost/${environment}`, l2InitFile, false);
```

Comment:
We should probably add the same `env.modify` for `DATABASE_PROVER_URL` and `TEST_DATABASE_PROVER_URL`

Reply:
Good idea. Added!